### ISSUE TITLE
Issue with DMM interface (upstream's #171)

### ISSUE THREAD

**Issue by gakada** [gakada/KanColleViewer#4]
https://github.com/Grabacr07/KanColleViewer/issues/171

For now can be fixed with:

    #spacing_top, #sectionWrap, #dmm-ntgnavi-renew, .ntgnavi-l, .area-menu, .menu-icon {

instead of

    #spacing_top, #sectionWrap {

in `OverrideStyleSheet` `value` in `KanColleViewer.exe.config`.

**Comment by Explorare** [comment]
Can't find `#spacing_top, #sectionWarp {` in KCV 4.1.5/ 4.1.3

```
<setting name="OverrideStyleSheet" serializeAs="String">
                <value>body {
    margin:0;
    overflow:hidden
}

#game_frame {
    position:fixed;
    left:50%;
    top:-16px;
    margin-left:-450px;
    z-index:1
}


.area-pickupgame
{
    display:none!important;
}</value>
```

**Comment by gakada** [comment]
Try adding it to `.area-pickupgame`, like this:

    .area-pickupgame, #dmm-ntgnavi-renew, .ntgnavi-l, .area-menu, .menu-icon {
    {
        display:none!important;
    }

**Comment by Explorare** [comment]
Thanks, it worked.

**Comment by gonewiththewind1995** [comment]
can you add the number of resource on top like this??
![top](https://cloud.githubusercontent.com/assets/15788184/11071798/3dcbc020-8815-11e5-813d-a4ae3c548a9a.png)

**Comment by gakada** [comment]
Well, I can. Did you added it yourself (in this screenshot)?

What is planned is choosable materials like in 4.x and up to 8 materials when there is enough space.

**Comment by gonewiththewind1995** [comment]
nope, that's KCV of my friend and and I took his pic. my friend get KCV from BossaGroove 
https://github.com/BossaGroove/KanColleViewer/releases

**Comment by gonewiththewind1995** [comment]
but I like your KCV than his, and i think we have enough space, you can replace "flamethrower" with abbreviation of "Instant build"

**Comment by gonewiththewind1995** [comment]
what the hell is this menu bar??, can u fix it in next version??
![cvcv](https://cloud.githubusercontent.com/assets/15788184/11185335/e1a1105e-8cae-11e5-891a-611fdeec6ab1.png)

**Comment by Explorare** [comment]
See https://github.com/gakada/KanColleViewer/issues/4#issuecomment-153984236

**Comment by gonewiththewind1995** [comment]
lol how can I fix this, I'm not IT-er :v

**Comment by Explorare** [comment]
Edit `KanColleViewer.exe.config` which should locate in your KCV folder.

Replace (Line 36)

```
.area-pickupgame
{
    display:none!important;
}
```

with

```
.area-pickupgame, #dmm-ntgnavi-renew, .ntgnavi-l, .area-menu, .menu-icon {
{
    display:none!important;
}
```

It seems that this problem has been fixed in the latest release c69d2a9 .

**Comment by gakada** [comment]
Fixed this in [r560](https://github.com/gakada/KanColleViewer/releases/tag/v3.8.2r560) (also added resources, though not choosable yet).
